include/revng-c/mlir/Dialect/Clift/IR/CliftAttributes.h

```diff
@@ -30,8 +30,8 @@ namespace mlir::clift {
 // discourse.llvm.org/t/custom-walk-and-replace-for-non-tablegen-types/74229
 // This is very brittle and it is very likely that it will change again in
 // future llvm releases
-class StructType
-  : public ::mlir::Attribute::AttrBase<StructType,
+class StructDefinition
+  : public ::mlir::Attribute::AttrBase<StructDefinition,
```

Comment:
This change was just caught by the regex I was using to replace the names all over the place. But after looking through it, I decided to keep it since it doesn't really make sense to keep it named `StructType` after this change-set. If nothing else, for the sake of consistency.

But right now, looking at his PR, it doesn't really make sense to call his version `StructDefinition` either - since at that point `Definition` is only ever used in ABI context.

So, I guess the logical thing to do is to resolve this at the rebase point. Luckily, it's pretty easy - all that's needed is to replace all non-llvm instances of `StructType` with `StructDefinition`.

Reply:
After the rebase I went with Lauri's naming (`XXXAttr`) here. To make his future rebase easier. And to not make the scope of this pr any bigger.